addons/web_editor/static/src/scss/web_editor.frontend.scss

```diff
@@ -32,46 +32,61 @@
     transform: scale(-1);
 }
 
+@include media-breakpoint-down(lg) {
+    @for $i from 0 through 12 {
+        .o_we_mobile_order_#{$i} {
+            order: $i;
+        }
+    }
+}
```

Comment:
I don't remember why I did it that way, but quickly checking I find that bootstrap 5 has 5 order-classes, while bootstrap 4 has 12. (+ one order-first and one order-last for both)
https://getbootstrap.com/docs/4.0/layout/grid/
> "Includes support for 1 through 12 across all five grid tiers."

https://getbootstrap.com/docs/5.0/layout/columns/
> "Includes support for 1 through 5 across all six grid tiers."

6 orders seems ok-ish for our use case but prone to generate errors (just copying one column in a 6-column design would generate one: probably a rare use case, but far from impossible).
And this also means adding `order-lg-0` everywhere there is an `order-x`, which looks to me like a potential source of bugs (although it would probably be contained, since the `order-lg-0` would only be added or removed with all `order-x` classes).

Reply:
We definitely have more that 6:
![image](https://github.com/odoo/odoo/assets/10338094/d2e5ea2f-994c-4ee3-8362-56dafcb1805f)

I guess it's 6 in default BS5 but 12 in our codebase to support BS4, see:
```scss
       "order": map-merge(
            map-get($utilities, "order"),
            (
                values: $order-array,
            ),
        ),
```

Sure one class instead of 2 (order-lg-0) would be better but it's the bootstrap way. I think it's better to avoid extra CSS rules when we can (even if it's only 12 small ones). Actually, your code would be better handling the removal of `order` classes when switching etc, since we should support bootstrap templates as much as possible.

So, in another PR after this, we should make the switch IMO :+1: